modules/lang-painless/src/main/java/org/elasticsearch/painless/Locals.java

```diff
@@ -21,14 +21,17 @@
 
 import org.elasticsearch.painless.Definition.Method;
 import org.elasticsearch.painless.Definition.MethodKey;
-import org.elasticsearch.painless.Definition.Type;
+import org.objectweb.asm.Type;
```

Comment:
Nit: I realize your IDE probably did this, but would you mind switching it back to ```org.elasticsearch.painless.Definition.Type;``` and list out the ```org.objectweb.asm.Type instead```?  It matches with the other imports already from Definition.